infrastructure/docker/docker-compose.yml

```diff
@@ -8,7 +8,10 @@ services:
         build: services/router
         volumes:
             - "/var/run/docker.sock:/var/run/docker.sock"
-        network_mode: "host"
```

Comment:
I would put the default (here) for linux, and add a `docker-composer.windows.yml` only when running on windows.

Reply:
That's not possible, as the merge does not allow to remove options.

Reply:
It should not be needed anymore since https://github.com/jolicode/docker-starter/pull/93

Could you try to remove it and try again?